Makefile

```diff
@@ -247,6 +247,10 @@ service-clients:
 	@printf $(COLOR) "Generate service clients..."
 	@go generate -run genrpcwrappers ./client/...
 
+server-interceptors:
+	@printf $(COLOR) "Generate server interceptors..."
+	@go generate -run genrpcserverinterceptors ./common/rpc/interceptor/logtags/...
```

Comment:
not sure you need the `-run` here, there's only one purpose to that package...

(for `service-clients` it's needed because there's some mockgen in that package)